examples/motoko_examples/quicksort/canisters/rust/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+mod quicksort;
+use ic_cdk::export::candid::Int;
+
+#[ic_cdk_macros::query]
```

Comment:
I didn't realize these didn't have benchmarks, I think because of the usefulness of these benchmarks that we should change them to update functions and benchmark them.